packages/flutter_test/test/window_test.dart

```diff
@@ -272,4 +273,7 @@ class FakeAccessibilityFeatures implements AccessibilityFeatures {
 
   @override
   final bool reduceMotion;
+
+  @override
```

Comment:
This override can be removed to resolve the analyzer failure.

Reply:
Done!